client/lib/importer/actions.js

```diff
@@ -159,7 +169,10 @@ export function resetImport( siteId, importerId ) {
 
 export function clearImport( siteId, importerId ) {
 	// We are done with this import session, so lock it away
-	lockImport( importerId );
+	Dispatcher.handleViewAction( {
+		type: IMPORTS_IMPORT_LOCK,
+		importerId,
+	} );
```

Comment:
Any reason why this call to `handleViewAction` doesn't have the action object defined separately, like [the others](https://github.com/Automattic/wp-calypso/pull/27396/files#diff-551d84e62db81f7f470d439d2e8241d6R148)?

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/27396/commits/c78debe31776d1c78eb2e204c9d98485a2c93076 🙂